src/inventory.py

```diff
@@ -17,8 +16,7 @@ def create_inventory_instance(backend="sqlite"):
         "{}Inventory".format(backend.title()))()
 
 
-@Singleton
-class Inventory():
+class Inventory:
     """
     Inventory singleton class which uses storage backends
```

Comment:
update the docstring to reflect it's not a singleton anymore but it's supposed to be a variable in global state.